api/v1alpha3/azurecluster_default.go

```diff
@@ -119,3 +124,30 @@ func generateNodeSecurityGroupName(clusterName string) string {
 func generateRouteTableName(clusterName string) string {
 	return fmt.Sprintf("%s-%s", clusterName, "node-routetable")
 }
+
+func (c *AzureCluster) generateControlPlaneIngressRules() IngressRules {
+
+	// default ingress rules for control plane nodes
+	return IngressRules{
+		&IngressRule{
+			Name:             "allow_ssh",
+			Description:      "Allow SSH",
+			Priority:         100,
+			Protocol:         SecurityGroupProtocolTCP,
+			Source:           to.StringPtr("*"),
+			SourcePorts:      to.StringPtr("*"),
+			Destination:      to.StringPtr("*"),
+			DestinationPorts: to.StringPtr("22"),
+		},
+		&IngressRule{
+			Name:             "allow_apiserver",
+			Description:      "Allow K8s API Server",
+			Priority:         101,
+			Protocol:         SecurityGroupProtocolTCP,
+			Source:           to.StringPtr("*"),
+			SourcePorts:      to.StringPtr("*"),
+			Destination:      to.StringPtr("*"),
+			DestinationPorts: to.StringPtr("6443"),
```

Comment:
for this you'll have to determine the port based on the Cluster object as discussed on slack

Reply:
you still going to remove the hard coding port here, right?

Reply:
Yes, still planning on fixing that bit